src/game_renderer/Program.cpp

```diff
@@ -114,3 +114,11 @@ GLint Program::getUniform(const string &name) const {
    }
    return uniform->second;
 }
+
+std::string Program::getName() const {
+   return progName;
+}
+
+void Program::setName(const string &name) {
```

Comment:
^ In that case id say its outside the scope of this review, since you just moved Program to another folder